src/createServer.js

```diff
@@ -1,3 +1,42 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { checkErrors } = require('./validation');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, 'http://localhost:5700');
```

Comment:
Hardcoding a host name and port number can be limiting and might not work in all environments. 
```suggestion
    const normalizedUrl = new URL(req.url, `http:${req.headers.host}`);
```